### ISSUE TITLE
Keyboard shortcut for 10th profile doesn't work

### ISSUE THREAD

**Issue by timheuer** [microsoft/terminal#826]
When you have more than 10 profiles, using the key shortcut CTRL+SHIFT+0 to launch the 10th profile doesn't work.  No-op.

Repro:
Load profiles.json
Copy one profile and paste it 11 more times (changing GUID)
Save profiles.json
Iterate through profile launching using CTRL+SHIFT+1 , 2, 3, etc. but when get to 0 it fails and doesn't launch that profile.

**Comment by mKay00** [comment]
I tried to fix it, since this seemed like an easy task. But everything is correct, so I tried to use another keybinding (STRG + 0) and then it works. Is there another shortcut that uses CTRL + SHIFT + 0?

**Comment by timheuer** [comment]
Yes I think something is handling it and I don't know what either...not clear from the terminal code if it is something there, but thanks for looking...wonder if in conhost somewhere...

**Comment by oising** [comment]
It looks like this is an issue that has surfaced before in other apps, and may be related to the Windows IME (input method editor) which probably has dibs on <kbd>ctrl</kbd>+<kbd>shift</kbd>+<kbd>0</kbd> with a global hook. 

Reference: https://support.microsoft.com/en-us/help/967893/input-method-editor-keyboard-shortcut-ctrl-shift-0-switches-the-input

**Comment by timheuer** [comment]
Good find @oising.  In Win10 the switching between IME inputs is default LeftShift+something (probably changed along the way), but my guess is Windows code still hanging on the ctrl+shift+0.

IMO, not worth trying to hunt down (actually felt odd to me that 0 was the '10th' anyway) and I'd actually recommend to the team that you just have the first 1-9 as the options.  This is, afterall, for launching and not tabbing through them (which is already accommodated)

**Comment by oising** [comment]
@timheuer Yep, that would be my guess too. There are reports in forums of people complaining about this with respect to Excel (2013 and 2016), as recently as last December. I'm assuming it's still hooking it, but not servicing it.

**Comment by mKay00** [comment]
Since I am already at it. I delete the shortcut for the 10th profile and make a PR.

**Comment by timheuer** [comment]
@mKay00 obviously make sure to link to the above URI in the PR notes to justify the change.  Seems righteous to me.
